packages/account/src/Components/verification-status-action-button/verification-status-action-button.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Button, ButtonLink, Text } from '@deriv/components';
+import style from './verifications-status-action-button.module.scss';
+
+type TButtonLink = {
+    onClick?: never;
+    to: string;
+};
+
+type TButton = {
+    onClick?: () => void;
+    to?: never;
+};
+
+type TVerificationStatusActionButton = { button_text: JSX.Element } & (TButtonLink | TButton);
```

Comment:
the goal of these types implementation is to prevent passing 'to' when 'onClick' is passed, and vice versa.
Because the component is design in the way, that 'onClick' and 'to' are not using simultaneously.

Reply:
```
import React from 'react';
import { Button, ButtonLink, Text } from '@deriv/components';
import style from './verifications-status-action-button.module.scss';

type TButtonLink = {
    type: 'link';
    to: string;
    onClick?: never;
};

type TButton = {
    type: 'button';
    onClick: () => void;
    to?: never;
};

type TVerificationStatusActionButton = { button_text: JSX.Element } & (TButtonLink | TButton);

const VerificationStatusActionButton = ({ button_text, type, onClick, to }: TVerificationStatusActionButton) => {
    return (
        <div className={''}>
            {type === 'link' ? (
                <ButtonLink className={style.actionButton} to={to}>
                    <Text size='xs' className={style.actionButton__text} as='p' weight='bold'>
                        {button_text}
                    </Text>
                </ButtonLink>
            ) : (
                <Button type='button' onClick={onClick} has_effect primary className={style.actionButton}>
                    <Text size='xs' weight='bold' as='p' color='general' className={style.actionButton__text}>
                        {button_text}
                    </Text>
                </Button>
            )}
        </div>
    );
};

export default VerificationStatusActionButton;
```

Reply:
we can do it, but it requires to pass additional props.
we don’t need it with my implementation